js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,24 @@
+
+function duplicateLetters(args) {
+  let chars = {};
+  let maxduplicates = 0;
```

Comment:
Make sure to use proper casing (camelCase) in variable naming.
`maxduplicates` -> `maxDuplicates`

`maximumFrequency` would have been a better variable name here.